SPECIFICATION.md

```diff
@@ -27,6 +27,7 @@ This document will use the following terms to define data types.
 4. **DATE**: a date, in [ISO 8601 (YYYYMMDD) Basic Notation](https://en.wikipedia.org/wiki/ISO_8601). Example: `20200201` is 1 February, 2020.
 5. **SHORTSTRING**: a sequence of US-ASCII characters which is limited to 8 bytes in length.
 6. **SHORTNUMERIC**: a **NUMERIC** with a maximum value of 9.
+7. **PHONE**: a E.164 formatted phone number as string. 
```

Comment:
Let's mention data length and character set here. US-ASCII, maximum 15 characters, right?

Reply:
Also, is Phone another CDC requirement?

Reply:
Nice! Yes. I am adding the limitation 

This is not technically a requirement for the CDC, but it can be a nice add on to apps that want to do some form of 2-step auth with the user. Instead of checking ID's, they can message you to make sure you are who you say you are.